### PR TITLE
Voltage angle limit support

### PR DESCRIPTION
**Pull request by annetill**
**Please check if the PR fulfills these requirements**

- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem?**

**What kind of change does this PR introduce?**

Support of voltage angle limit for AC security analysis only. Impossible easily for DC security analysis... without thinking about a refactoring of DC sensitivity analysis or implementing a "slow" mode of DC security analysis.

**What is the current behavior?**

**What is the new behavior (if this is a feature change)?**

**Does this PR introduce a breaking change or deprecate an API?**

- [ ] The *Breaking Change* or *Deprecated* label has been added
- [ ] The migration guide has been updated in the github wiki *(What changes might users need to make in their application due to this PR?)*

**Other information**:

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -48,7 +48,7 @@
         <slf4jtoys.version>1.6.3</slf4jtoys.version>
         <asciitable.version>0.3.2</asciitable.version>
 
-        <powsybl-core.version>6.0.0</powsybl-core.version>
+        <powsybl-core.version>6.0.1</powsybl-core.version>
```
Comment (geofjamg) [inline pom.xml:51]: Not in this PR!

File: src/test/java/com/powsybl/openloadflow/sa/OpenSecurityAnalysisTest.java [hunk 2]
```diff
@@ -2240,4 +2241,39 @@ void testWithShuntVoltageControlContingency() {
...
+    void testVoltageAngleLimit() {
+        Network network = EurostagTutorialExample1Factory.create();
+        Line line = network.getLine("NHV1_NHV2_1");
+        network.newVoltageAngleLimit().setId("val").from(line.getTerminal1()).to(line.getTerminal2()).setHighLimit(7.0).setLowLimit(-7.0).add();
+        LoadFlow.run(network);
+        LoadFlowAssert.assertAngleEquals(0.0, line.getTerminal1().getBusView().getBus());
+        LoadFlowAssert.assertAngleEquals(-3.506413, line.getTerminal2().getBusView().getBus());
+        network.getLine("NHV1_NHV2_2").getTerminal1().disconnect();
+        network.getLine("NHV1_NHV2_2").getTerminal2().disconnect();
+        LoadFlow.run(network);
```
Comment (geofjamg) [inline src/test/java/com/powsybl/openloadflow/sa/OpenSecurityAnalysisTest.java:2255]: Use loadFlowProvider instead that is configured to use dense matrices

File: src/test/java/com/powsybl/openloadflow/sa/OpenSecurityAnalysisTest.java [hunk 3]
```diff
@@ -2240,4 +2241,39 @@ void testWithShuntVoltageControlContingency() {
         PostContingencyResult postContingencyResult = getPostContingencyResult(result, "contingency");
         assertEquals(390.0, postContingencyResult.getNetworkResult().getBusResult("b4").getV(), 0.001);
     }
+
+    @Test
+    void testVoltageAngleLimit() {
+        Network network = EurostagTutorialExample1Factory.create();
+        Line line = network.getLine("NHV1_NHV2_1");
+        network.newVoltageAngleLimit().setId("val").from(line.getTerminal1()).to(line.getTerminal2()).setHighLimit(7.0).setLowLimit(-7.0).add();
+        LoadFlow.run(network);
```
Comment (geofjamg) [inline src/test/java/com/powsybl/openloadflow/sa/OpenSecurityAnalysisTest.java:2250]: Same

File: src/test/java/com/powsybl/openloadflow/sa/OpenSecurityAnalysisTest.java [hunk 4]
```diff
@@ -2240,4 +2241,39 @@ void testWithShuntVoltageControlContingency() {
...
+        LoadFlowAssert.assertAngleEquals(0.0, line.getTerminal1().getBusView().getBus());
+        LoadFlowAssert.assertAngleEquals(-7.498849, line.getTerminal2().getBusView().getBus());
+
+        network.getLine("NHV1_NHV2_2").getTerminal1().connect();
+        network.getLine("NHV1_NHV2_2").getTerminal2().connect();
+        List<Contingency> contingencies = List.of(new Contingency("contingency", List.of(new BranchContingency("NHV1_NHV2_2"))));
+        SecurityAnalysisResult result = runSecurityAnalysis(network, contingencies, Collections.emptyList(), new SecurityAnalysisParameters());
+        LimitViolation limit = result.getPostContingencyResults().get(0).getLimitViolationsResult().getLimitViolations().get(0);
+        assertEquals(LimitViolationType.LOW_VOLTAGE_ANGLE, limit.getLimitType());
+        assertEquals(-7.498849, limit.getValue(), 10E-6);
```
Comment (geofjamg) [inline src/test/java/com/powsybl/openloadflow/sa/OpenSecurityAnalysisTest.java:2265]: Use the constant defined in LoadFlowAssert

File: src/test/java/com/powsybl/openloadflow/sa/OpenSecurityAnalysisTest.java [hunk 5]
```diff
@@ -2240,4 +2241,39 @@ void testWithShuntVoltageControlContingency() {
...
+
+        network.getVoltageAngleLimit("val").remove();
+        network.newVoltageAngleLimit().setId("val").from(line.getTerminal2()).to(line.getTerminal1()).setHighLimit(7.0).setLowLimit(-7.0).add();
+        SecurityAnalysisResult result2 = runSecurityAnalysis(network, contingencies, Collections.emptyList(), new SecurityAnalysisParameters());
+        LimitViolation limit2 = result2.getPostContingencyResults().get(0).getLimitViolationsResult().getLimitViolations().get(0);
+        assertEquals(LimitViolationType.HIGH_VOLTAGE_ANGLE, limit2.getLimitType());
+        assertEquals(7.498849, limit2.getValue(), 10E-6);
```
Comment (geofjamg) [inline src/test/java/com/powsybl/openloadflow/sa/OpenSecurityAnalysisTest.java:2272]: Also

File: pom.xml (hunk 1 above)
Comment (annetill) [reply pom.xml:51]: See #881

**Review by geofjamg — APPROVED** [review]

File: src/test/java/com/powsybl/openloadflow/sa/OpenSecurityAnalysisTest.java (hunk 2 above)
Comment (geofjamg) [reply src/test/java/com/powsybl/openloadflow/sa/OpenSecurityAnalysisTest.java]: Done

File: src/test/java/com/powsybl/openloadflow/sa/OpenSecurityAnalysisTest.java (hunk 3 above)
Comment (geofjamg) [reply src/test/java/com/powsybl/openloadflow/sa/OpenSecurityAnalysisTest.java]: Done

File: src/test/java/com/powsybl/openloadflow/sa/OpenSecurityAnalysisTest.java (hunk 4 above)
Comment (geofjamg) [reply src/test/java/com/powsybl/openloadflow/sa/OpenSecurityAnalysisTest.java]: Done

File: src/test/java/com/powsybl/openloadflow/sa/OpenSecurityAnalysisTest.java (hunk 5 above)
Comment (geofjamg) [reply src/test/java/com/powsybl/openloadflow/sa/OpenSecurityAnalysisTest.java]: Done

**Review by jeandemanged — APPROVED** [review]

**Pull request merged**